docbook/Admin_Guide/en-US/config/html.xml

```diff
@@ -98,48 +98,31 @@
 				</warning>
 			</listitem>
 		</varlistentry>
-
-		<varlistentry>
-			<term>$g_bottom_include_page</term>
-			<listitem>
-				<para>Specifies a file to be included at the bottom of each page.
-					It can be used e.g. for company branding, to include Google
-					Analytics script, etc.
-				</para>
-			</listitem>
-		</varlistentry>
 		<varlistentry>
 			<term>$g_top_include_page</term>
 			<listitem>
-				<para>Specifies a file to be included at the top of each page.
+				<para>Absolute path to the top include file.
 					It can be used e.g. for company branding.
 				</para>
-				<para>If a file is supplied, the logo specified by
-					<literal>$g_logo_image</literal> (see
-					<xref linkend="admin.config.display" />) will not be shown,
-					and the include file will have to handle display of the logo.
-					To do so you can use the <literal>html_print_logo()</literal>
-					API function, which will display the logo with an URL link if
-					one has been specified in <literal>$g_logo_url</literal>
-				</para>
-				<para>Example top include PHP file with logo and centered page title:
+				<para>Example include file with logo and centered page title:
```

Comment:
The new example does not actually display a logo anymore. The `html_print_logo();` call should be added back.

Reply:
Indeed.